services/brig/src/Brig/User/Auth.hs

```diff
@@ -143,8 +148,17 @@ revokeAccess u pw cc ll = do
 --------------------------------------------------------------------------------
 -- Internal
 
+catchSuspendInactiveUser :: UserId -> e -> ExceptT e AppIO ()
+catchSuspendInactiveUser uid errval = do
+  mustsuspend <- lift $ mustSuspendInactiveUser uid
+  Log.debug . Log.msg $ show (uid, mustsuspend)
```

Comment:
Can we instead use the `user` field here to log the uuid

Reply:
yes.